src/App.jsx

```diff
@@ -14,33 +16,86 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortByAlphabet = () => {
+    setGoods([...goods].sort((a, b) => a.localeCompare(b)));
+    setSortField('light');
+  };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const sortByLength = () => {
+    setGoods([...goods].sort((a, b) => a.length - b.length));
+    setSortField('length');
+  };
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  const sortByReverse = () => {
+    setGoods([...goods].reverse());
+    setSortField('reverse');
+  };
```

Comment:
You can combine all of these functions into one and pass goods, sortType, and reverse state to it and based on them you will calculate a new goods array